app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -200,9 +200,19 @@ def get_network_ports
   end
 
   def get_ec2_floating_ips_and_ports
-    instances = @aws_ec2.instances.select { |instance| instance.network_interfaces.blank? }
-    process_collection(instances, :network_ports) { |instance| parse_network_port_inferred_from_instance(instance) }
-    process_collection(instances, :floating_ips) { |instance| parse_floating_ip_inferred_from_instance(instance) }
+    instances = @aws_ec2.instances
+    instances = instances.select do |instance|
+      interfaces = instance.network_interfaces
+      interfaces.blank? || interfaces.size.zero?
```

Comment:
I don't understand this... `.blank?` (normally) checks for empty collections, so is there a time when the are 0 items, and `blank?` returns `false`?

Reply:
The reason is that its not kind of `Enumerable`. 
This fix is intentional, without it there was few related issues.